daffodil-io/src/main/scala/org/apache/daffodil/io/ByteBufferDataInputStream.scala

```diff
@@ -91,7 +90,13 @@ object ByteBufferDataInputStream {
       case _ => {
         // copy the contents of the stream into an array of bytes
         val bos = new ByteArrayOutputStream
-        IOUtils.copy(in, bos)
```

Comment:
BTW I think this change was because i needed to breakpoint on the read or write when debugging.